client/src/pages/NeedHelp.js

```diff
@@ -72,14 +73,18 @@ const Step2 = (props) => {
       </AnswerButton>
     </WizardStep>
   );
-};
-
+}; 
 const Step3 = (props) => {
   const [email, setEmail] = useState("");
   const onChange = (evt) => setEmail(evt);
-  const onSubmit = () => {
+  const onSubmit = (evt) => {
+    evt.preventDefault();
+    if (!validateEmail(email)) {
+      Toast.fail("Invalid email address!", 2.0);
```

Comment:
I will double check with designers but I think my idea was to have client-side validation handled by form components, like adding an error state
And network requests errors should come back with a pop-up... for example "user already exists" cannot be hanvled by the form itself.